src/SelectLocation.js

```diff
@@ -112,10 +118,10 @@ export default class SelectLocation extends Component {
   signClick = id => {
     // set state.sign for Popup parameters in render, center map to clicked sign
     const clickedSign = this.state.signs.find(sign => sign.id === id);
-    const newCenter = [clickedSign.lng, clickedSign.lat];
+    // const newCenter = [clickedSign.lng, clickedSign.lat];
     this.setState({
-      activeSign: clickedSign,
-      center: newCenter
+      activeSign: clickedSign
+      // center: newCenter
```

Comment:
I didn't understand what this changed from your comment but I trust your judgement if you think its an improvement. Send a screenshot of what changed if you want more feedback on it.

Let's go ahead and delete the commented out lines once you feel good about it.

Reply:
The old behavior was that the map would re-center on a Marker when it was clicked to see the details pop-up. It felt jarring after using it for a while, and it could get annoying on mobile since the map window can feel smaller (easier to lose your place). The new behavior is the map does not re-center when a Marker is clicked.